website/scripts/should-build.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
+# Copyright (c) HashiCorp, Inc.
```

Comment:
I think this is okay to merge in since the digital team should be adding this to their management and we can flag it during review if the header is trying to be removed.